src/main/java/uk/gov/mca/beacons/api/services/GetPersonByIdService.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.mca.beacons.api.services;
+
+import java.util.UUID;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.mca.beacons.api.domain.PersonType;
+import uk.gov.mca.beacons.api.gateways.PersonGateway;
+import uk.gov.mca.beacons.api.jpa.entities.Person;
+
+@Service
+@Transactional
+public class GetPersonByIdService {
+
+  private final PersonGateway personGateway;
+
+  public GetPersonByIdService(PersonGateway personGateway) {
+    this.personGateway = personGateway;
+  }
+
+  public Person execute(UUID id, PersonType personType) {
```

Comment:
I would actually have this return an `Optional<Owner>` rather than returning `null` and having the controller to check whether the object is null. This will allow you to do something along the lines of: `Optional.isPresent()`.  See docs on this: https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html#isPresent--

JPA repositories can return `Optional<T>` if you define that as the return type on the repository interface: https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#appendix.query.return.types

Reply:
If you change this service to `GetOwnerByIdService` then you can remove the `PersonType` which also reduces the complexity of this class and serves just the one purpose of getting an owner rather than an owner or emergency contact.